projects/manage-supervision-and-delius/src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/SenetenceIntegrationTest.kt

```diff
@@ -0,0 +1,51 @@
+package uk.gov.justice.digital.hmpps
```

Comment:
Typo - Rename to SentenceIntegrationTest

Reply:
done